docs/Deploy/Introduction.md

```diff
@@ -16,11 +16,7 @@ The `fleetctl` binary is the CLI interface which allows management of your deplo
 
 Both binaries are available for download from our [repo](https://github.com/fleetdm/fleet/releases).
 
-## Fleet vs Fleet Sandbox
-
-If you'd like to try Fleet on your laptop, we recommend [Fleet Sandbox](https://fleetdm.com/try-fleet/register).
-
-If you want to enroll real hosts or deploy to a more scalable environment, we recommend [deploying Fleet to a server](https://fleetdm.com/docs/deploy/deploy-fleet-on-centos).
+> If you want to enroll real hosts or deploy to a more scalable environment, we recommend [deploying Fleet to a server](https://fleetdm.com/docs/deploy/deploy-fleet-on-centos).
```

Comment:
I think we can delete this part too; with the sandbox option gone, you're always here to deploy Fleet to a server, so no need to explicitly mention it.
```suggestion
```